src/material/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -92,6 +95,13 @@ export abstract class _MatTabNavBase
   /** Theme color of the nav bar. */
   @Input() color: ThemePalette = 'primary';
 
+  /**
+   * Associated tab panel controlled by the nav bar. If not provided, then the nav bar
+   * follows the ARIA link / navigation landmark pattern. If provided, it follows the
+   * ARIA tabs design pattern.
+   */
+  @Input() panel?: MatTabNavPanel;
```

Comment:
The input name might need to be prefixed since the component uses an attribute selector.

Reply:
Do you mind clarifying? Not sure what change you're requesting, sorry.

Reply:
Sorry, what I meant is that the component currently uses an attribute selector which means that it could have other directives attached to it so the `panel` name could conflict with something else. I was suggesting changing the name to something like `tabPanel`.

Reply:
Done.